npm/debugTools/dataplane/iptable.go

```diff
@@ -0,0 +1,211 @@
+package dataplane
+
+import (
+	"fmt"
+	"strings"
+)
+
+type Iptables struct {
+	name   string
+	chains map[string]*IptablesChain
+}
+
+// NewIptables is the constructor for iptables
+func NewIptables(name string, chains map[string]*IptablesChain) *Iptables {
+	iptable := &Iptables{}
+	iptable.name = name
+	iptable.chains = chains
+	return iptable
+}
+
+func (t *Iptables) Name() string {
+	return t.name
+}
+
+func (t *Iptables) SetName(name string) {
+	t.name = name
+}
+
+func (t *Iptables) Chains() map[string]*IptablesChain {
+	return t.chains
+}
+
+func (t *Iptables) SetChains(chain_m map[string]*IptablesChain) {
+	t.chains = chain_m
+}
```

Comment:
[I left this feedback on your other PR](https://github.com/Azure/azure-container-networking/pull/951#discussion_r675148161); don't do this Java style constructor/getter/setter fluff in Go. If the struct fields need to be directly accessed outside of the package, make them public and initialize the struct directly.

Reply:
this bit of code was my doing ! Thanks for the suggestion.